mite/cli/test.py

```diff
@@ -152,7 +152,9 @@ def test_scenarios(test_name, opts, scenarios, config_manager):
     loop.run_until_complete(asyncio.wait(coroutines, return_when=asyncio.FIRST_COMPLETED))
     # Run one last report before exiting
     controller.report(receiver.recieve)
-    has_error = http_stats_output is not None and http_stats_output.error_total > 0
+    has_error = http_stats_output is not None and http_stats_output.error_total > int(
+        opts.get("--max-errors-threshold")
+    )
```

Comment:
Would it make sense to detect the case where 1 > `opts.get("--max-errors-threshold")` >= 0 and, in that case, treat the value as a percentage of total requests instead?  That's an additional feature on top of what you implemented (so it's not a requirement to merge the PR), but it seems like it would be useful...

Reply:
I had a look if we already have a variable for the total amount of journeys ran but looks it is not there. I like the idea though, I'll write it down as an improvement to add